plugin/pkg/scheduler/api/v1/types.go

```diff
@@ -100,3 +108,52 @@ type LabelPreference struct {
 	// If false, higher priority is given to nodes that do not have the label
 	Presence bool `json:"presence"`
 }
+
+// Holds the parameters used to communicate with the extender. If a verb is unspecified/empty,
+// it is assumed that the extender chose not to provide that extension.
+type ExtenderConfig struct {
+	// URLPrefix at which the extender is available
+	URLPrefix string `json:"urlPrefix"`
+	// Verb for the filter call, empty if not supported
+	FilterVerb string `json:"filterVerb,omitempty"`
+	// Verb for the prioritize call, empty if not supported
+	PrioritizeVerb string `json:"prioritizeVerb,omitempty"`
+	// EnableHttps specifies whether https should be used to communicate with the extender
+	EnableHttps bool `json:"enableHttps,omitempty"`
+	// TLSConfig specifies the transport layer security config
+	TLSConfig *client.TLSClientConfig `json:"tlsConfig,omitempty"`
+	// HTTPTimeout specifies the timeout duration for a call to the extender. Filter timeout
+	// fails the scheduling of the pod. Prioritize timeout is ignored, k8s/other extenders
+	// priorities are used to select the node.
+	HTTPTimeout time.Duration `json:"httpTimeout,omitempty"`
+}
```

Comment:
Just starting to look, so ignore the dumb question as I keep looking, but why does `plugin/pkg/scheduler/api/types.go::ExtenderConfig` have a `Weight` but this doesn't? 

/me is still trying to figure out what the `Weight` means over there...